webapp/channels/src/components/latex_inline/latex_inline.test.tsx

```diff
@@ -1,21 +1,28 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {shallow} from 'enzyme';
+import {render} from '@testing-library/react';
 import React from 'react';
+import {act} from 'react-dom/test-utils';
 
 import LatexInline from 'components/latex_inline/latex_inline';
 
-describe('components/LatexBlock', () => {
+import {withIntl} from 'tests/helpers/intl-test-helper';
+
+describe('components/LatexInline', () => {
     const defaultProps = {
         content: 'e^{i\\pi} + 1 = 0',
         enableInlineLatex: true,
     };
 
     test('should match snapshot', async () => {
-        const wrapper = shallow(<LatexInline {...defaultProps}/>);
-        await import('katex'); //manually import katex
-        expect(wrapper).toMatchSnapshot();
+        let container;
+
+        await act(async () => {
```

Comment:
I am curious. Why `act` is needed?

Reply:
For the state change to take place correctly. Similar logic can be found in latex_block

Reply:
Makes sense. Thank you for the explanation!

Reply:
act is used internally in testing-library/react so we might not need this

Reply:
`renderWithContext` uses `render` intenally which uses `act` internally, while `screen.findBy` are just selector function which doesnt wrap act.
However it seems that this behavior is due to async import of the module in `latex_inline.tsx` so here act is justified